src/test/proto/custom_field_options.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package custom_field_options;
+
+option go_package = "pb-go/custom_field_options";
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.FieldOptions {
+  bool key = 50002;
+  bool required = 50003;
```

Comment:
The tests use two field options for some of the fields. All options used in tests need to be valid, so add a third option here:

```
bool internal = 50004;
```